docs/report/028.ipynb

```diff
@@ -367,7 +367,7 @@
    },
    "source": [
     ":::{warning}\n",
-    "Not all final state in the `description`s can be programmatically deciphered as individual particles. One could try to use [regular expressions](https://docs.python.org/3/howto/regex.html), but it's hard to cover all cases. Consider for instance the following case which contains $S$ and $D$ waves.\n",
+    "Not all final state in the `descriptions` can be programmatically deciphered as individual particles. One could try to use [regular expressions](https://docs.python.org/3/howto/regex.html), but it's hard to cover all cases. Consider for instance the following case which contains $S$ and $D$ waves.\n",
```

Comment:
`description`s was used to emphasise that `description` is an attribute. 0070f8f makes it clearer with a link